### PR TITLE
Added babylon 5, closes #87

### LINKED ISSUE #87: Add `Babylon 5`?
Suggested on Reddit:
https://www.reddit.com/r/scifi/comments/380izr/scifi_worth_consuming/crrdsw7

### PR DESCRIPTION
**Pull request by RichardLitt**
Because.

-

#87

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1039,6 +1039,16 @@ This is a classic film, and well worth watching if only for the incredible and i
 
 ## TV Series
 
+#### [Babylon 5](http://www.imdb.com/title/tt0105946/) [8.2]
+
+Babylon 5's special effects left a lot to be desired, but the plot was amazing. - [@elektrovert](https://github.com/elektrovert)
+
+The CGI hasn't aged well--but the show's strength lies in character development and (on a meta level) the pre-planned structure which is fairly unique for TV series. On the other hand, it's a polarizing show... So my vote is a conditional yes :) - [@mihailim](https://github.com/mihailim)
```
Comment (sindresorhus) [inline readme.md]: `--` => `—`

Comment (sindresorhus) [inline readme.md]: Fixed it for ya.

**Comment by sindresorhus** [comment]
Thanks Richard :)

**Comment by RichardLitt** [comment]
Thanks @sindresorhus!

**Pull request closed without merging**